core/prelude/operators/as.carbon

```diff
@@ -12,3 +16,54 @@ interface ImplicitAs(Dest:! type) {
   // TODO: extend As(Dest);
   fn Convert[self: Self]() -> Dest;
 }
+
+// TODO: These impls should live with Core.Int, but currently that's a builtin
+// not a class type, so there is no other library these can go in.
```

Comment:
Had you considered `core/prelude/types/int.carbon`, similar to the `i32.carbon`?